bitrise.yml

```diff
@@ -234,6 +234,7 @@ workflows:
             echo "-- unzip -- "
             exec unzip "${BITRISE_ARTIFACT_PATHS}"
     - xcode-test-without-building@0:
+        run_if: '{{getenv "RUN_UI_TESTS" | eq "Run_UI_Tests"}}'
```

Comment:
Adding this to not start the tests if a step failed above or running tests is not necessary due to the files changed in the PR